samples/audio/sof/boards/intel_adsp_cavs15.conf

```diff
@@ -6,3 +6,4 @@ CONFIG_LP_MEMORY_BANKS=2
 CONFIG_HP_MEMORY_BANKS=8
 CONFIG_PERFORMANCE_COUNTERS=y
 CONFIG_COMP_SRC_SMALL=y
+CONFIG_RESET_VECTOR_IN_BOOTLOADER=n
```

Comment:
I think add this configuration to common config is better than samples(like boards/xtensa/intel_adsp_cavs15/intel_adsp_cavs15_defconfig)

Reply:
@KangJianX that doesn't work because then it's overwritten by samples/audio/sof/prj.conf. What we could do though is remove that setting from prj.conf and specify it in each boards/xtensa/intel_adsp_cavs*/intel_adsp_cavs*_defconfig individually where we need the non-default value. Would that be better?